packages/components/webpack.config.js

```diff
@@ -10,33 +10,39 @@ const is_release = process.env.NODE_ENV === 'production' || process.env.NODE_ENV
-        popover            : path.resolve(__dirname, 'src', 'components/popover/index.js'),
-        'radio-group'      : path.resolve(__dirname, 'src', 'components/radio-group/index.js'),
-        table              : path.resolve(__dirname, 'src', 'components/table/index.js'),
-        tabs               : path.resolve(__dirname, 'src', 'components/tabs/index.js'),
-        'themed-scrollbars': path.resolve(__dirname, 'src', 'components/themed-scrollbars/index.js'),
-        'toggle-switch'    : path.resolve(__dirname, 'src', 'components/toggle-switch/index.js'),
+        accordion                       : 'Components/accordion',
+        autocomplete                    : 'Components/autocomplete',
+        button                          : 'Components/button',
+        'button-toggle'                 : 'Components/button-toggle',
+        checkbox                        : 'Components/checkbox',
+        counter                         : 'Components/counter',
+        dialog                          : 'Components/dialog',
+        drawer                          : 'Components/drawer',
+        dropdown                        : 'Components/dropdown',
+        'field-error'                   : 'Components/field-error',
+        'file-dropzone'                 : 'Components/file-dropzone',
+        icon                            : 'Components/icon',
+        'icon/js/icons'                 : 'Components/icon/icons.js',
+        input                           : 'Components/input',
+        label                           : 'Components/label',
+        loading                         : 'Components/loading',
+        modal                           : 'Components/modal',
+        money                           : 'Components/money',
+        'password-input'                : 'Components/password-input',
+        'password-meter'                : 'Components/password-meter',
+        popover                         : 'Components/popover',
+        'radio-group'                   : 'Components/radio-group',
+        table                           : 'Components/table/index.js',
+        tabs                            : 'Components/tabs',
+        'themed-scrollbars'             : 'Components/themed-scrollbars',
+        'toggle-switch'                 : 'Components/toggle-switch',
+        'vertical-tab'                  : 'Components/vertical-tabs/vertical-tab',
+        'vertical-tab-content-container': 'Components/vertical-tabs/vertical-tab-content-container',
+        'vertical-tab-header'           : 'Components/vertical-tabs/vertical-tab-header',
+        'vertical-tab-header-title'     : 'Components/vertical-tabs/vertical-tab-header-title',
+        'vertical-tab-headers'          : 'Components/vertical-tabs/vertical-tab-headers',
+        'vertical-tab-layout'           : 'Components/vertical-tabs/vertical-tab-layout',
+        'vertical-tab-wrapper'          : 'Components/vertical-tabs/vertical-tab-wrapper',
```

Comment:
Do all of these need to be exported? Are all of them interfaces? And can we export them in the "built-in" components format like the `Table` component?

Reply:
That makes sense.